Source/Core/DolphinQt/CMakeLists.txt

```diff
@@ -606,6 +606,47 @@ else()
   install(TARGETS dolphin-emu RUNTIME DESTINATION ${bindir})
 endif()
 
+if (CMAKE_SYSTEM_NAME STREQUAL "Linux" AND STEAM)
+  # Set that we want ORIGIN in FLAGS.
+  # We also want RPATH, not RUNPATH, so disable the new tags.
+  target_link_options(dolphin-emu
+  PRIVATE
+    LINKER:-z,origin
+    LINKER:--disable-new-dtags
+  )
+
+  # For Steam Runtime builds, our Qt shared libraries will be in a "lib" folder.
+  set_target_properties(dolphin-emu PROPERTIES
+    BUILD_WITH_INSTALL_RPATH true
+    INSTALL_RPATH "\$ORIGIN/lib"
+  )
+
+  add_custom_command(TARGET dolphin-emu POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E make_directory "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/lib"
+    COMMAND cp -P "${QT_DIR}/../../*.so*" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/lib"
+    COMMAND ${CMAKE_COMMAND} -E copy_directory "${QT_DIR}/../../../plugins/platforms" "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/platforms"
```

Comment:
There should be a better way to find the plugins path. This is failing on system libs for ubuntu, and rolls-back the compilation. The path for Ubuntu 20.04 for example is: `"${QT_DIR}/../../qt/plugins/platforms"`
And why copy a blank `qt.conf` file. Where is it being set?

Reply:
Building with system libs is not expected to work, since by definition that wouldn't be a steam runtime build.

Reply:
Nicely spotted, see my bug report: https://bugs.dolphin-emu.org/issues/12946

Reply:
Should be fixed by #10728, steamrt builds shouldn't have defaulted to ON. Thanks for the reports.